lxd/networks.go

```diff
@@ -1794,6 +1795,10 @@ func (n *network) Start() error {
 				dnsmasqCmd = append(dnsmasqCmd, []string{"-s", "__internal", "-S", "/__internal/"}...)
 				dnsmasqCmd = append(dnsmasqCmd, []string{"-S", fmt.Sprintf("/%s/%s#1053", dnsDomain, dnsClusteredAddress)}...)
 				dnsmasqCmd = append(dnsmasqCmd, fmt.Sprintf("--dhcp-option=15,%s", dnsDomain))
+
+				if overlaySubnet != nil {
```

Comment:
I don't think it's possible to have dnsClustered set to true and have overlaySubnet be nil